testing/src/cypress/custom-command/institution/common-helper-functions.cy.ts

```diff
@@ -49,4 +51,8 @@ export default class InstitutionHelperActions {
     cy.visit(URL);
     institutionCustomCommand.loginWithCredentials(USERNAME, PASSWORD);
   }
+
+  getUniqueId() {
```

Comment:
I am not following this. A "guid" can be considered unique a part of the "guid" maybe not.
What is the intention of this method and why is it using only a partial "guid"?

Reply:
updated to use the complete uuid